.github/workflows/run_mmperf.yml

```diff
@@ -51,10 +52,17 @@ jobs:
           # https://github.com/nektos/act where there will be more.
           git log --oneline --graph --max-count=3
           ./build_tools/github_actions/configure_ci.py
+      - name: "Checkout out SHARK-Runtime"
```

Comment:
I still think it's weird that we're "using" an IREE commit and checking it out above for scripting and then using some other commit of IREE from a downstream fork... Do we actually want to be testing against the SHARK version? Should we be testing against IREE proper, since that's actually where this workflow lives? It is certainly possible to point the shark tank submodule at upstream IREE, if that's what we want to do. Just requires including the IREE repo URL in the fetch command.

Reply:
Updated to point to the IREE repo URL and use the same SHA the workflow is using.